completions/bash/cou_bash_completion

```diff
@@ -20,6 +20,16 @@ _subcommand_args() {
     esac
 }
 
+_sub_subcommand_args() {
+    local cmd=$1
+    case "${cmd}" in
+        hypervisors)
+            opts="--machine --availability-zone"
+            child_commands=""
+            ;;
+    esac
```

Comment:
We need to also add targets for `control-plane` and `data-plane` sub-commands because they still need to should some common opts.